lib/roadmap.rb

```diff
@@ -0,0 +1,40 @@
+module Roadmap
+    def get_roadmap(id = get_roadmap_id, refetch=false) # return roadmap for given roadmap id. default is the current roadmap of the user
+        if !refetch && id == get_roadmap_id && @my_roadmap
+            return @my_roadmap
+        end
+        headers = {
+            "Content-Type" => "application/json",
+            "authorization" => @auth_token
+        }
+        response = self.class.get("/roadmaps/#{id}", headers: headers)
+        body_text = response.body
+        @roadmap = JSON.parse(body_text)
+        @roadmap
```

Comment:
This is redundant. The last value of the method is what is returned (in this case @roadmap).